source/types/lowercase-keys.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'lowercase-keys' {
+	export default function(object: Record<string, string | string[] | number | boolean>): Record<string, string | string[] | number | boolean>;
```

Comment:
This package ships with its own type definitions now: https://github.com/sindresorhus/lowercase-keys/commits/master

Reply:
Is the version published on npm now? That way, I can just do a quick update and delete the file

Reply:
Yes, just update to version 2.0.0.